lib/app/modules/timer/views/timer_view.dart

```diff
@@ -448,6 +449,11 @@ class TimerView extends GetView<TimerController> {
                 ),
               ),
               // LanguageMenu(),
+              Divider(
+                color: Colors.white,
```

Comment:
divider should change it's colour on theme, currently it might not be visible in light mode

Reply:
Honestly, I think the divider looks off (since we have only two options). We should just remove it.

Reply:
yes we can remove it, old one was good too